include/Foundation/NSFileManager.h

```diff
@@ -18,6 +18,7 @@
 #import <Foundation/FoundationExport.h>
 #import <Foundation/NSObject.h>
 #import <Foundation/NSPathUtilities.h>
+#import <Foundation/NSDictionary.h>
```

Comment:
Ok. Thanks.  I guess I find it amazing that this is the first apianalyzer failure of this kind that I can remember.

---
In reply to: [111599749](https://github.com/Microsoft/WinObjC/pull/2520#discussion_r111599749) [](ancestors = 111599749)